bitmind/synthetic_image_generation/image_annotation_generator.py

```diff
@@ -108,7 +139,12 @@ def process_image(
 
         original_dimensions = image_info['image'].size
         start_time = time.time()
-        annotation = self.generate_annotation(image_index, dataset_name, image_info['image'], original_dimensions, resize, verbose)
+        annotation = self.generate_annotation(image_index,
+                                              dataset_name,
+                                              image_info['image'],
+                                              original_dimensions,
+                                              resize,
+                                              verbose)
         time_elapsed = time.time() - start_time
 
         if annotation == -1:
```

Comment:
@dylanuys Updated `generate_annotation` to throw an exception and return `None` instead of `-1` if failure to generate. `process_image` also updated to catch `None`s.